packages/webview_flutter/webview_flutter_wkwebview/lib/src/web_kit/web_kit.dart

```diff
@@ -46,6 +46,51 @@ enum WKAudiovisualMediaType {
   all,
 }
 
+/// Types of data that websites store.
+///
+/// See https://developer.apple.com/documentation/webkit/wkwebsitedatarecord/data_store_record_types?language=objc.
+enum WKWebsiteDataTypes {
+  /// Cookies.
+  ///
+  /// See https://developer.apple.com/documentation/webkit/wkwebsitedatatypecookies?language=objc.
```

Comment:
Nit: You could drop all of these links. There's no useful information in any of these pages; only the top-level link that you have in the class-level comment is actually useful.